docs/references.bib

```diff
@@ -635,3 +635,12 @@ @article{kaposi2021polynomial
   journal={arXiv preprint arXiv:2109.04528},
   year={2021}
 }
+
+@misc{bulmer2022threshold,
```

Comment:
Why `misc` and not `article`? I think the `journal` field might need to be added then (like above).
```suggestion
@article{bulmer2022threshold,
```

Reply:
I think is because we typically use google scholar to get bibtex entry, but google takes a few days to update. This entry was likely generated from arXiv which for some reason uses `@misc` instead of `@article`. Anyway, it has been updated.